Looms.PoS/Looms.PoS.Domain/Interfaces/IReservationsRepository.cs

```diff
@@ -0,0 +1,12 @@
+using Looms.PoS.Domain.Daos;
+
+namespace Looms.PoS.Domain.Interfaces;
+
+public interface IReservationsRepository
```

Comment:
[JsonConverter(typeof(JsonStringEnumConverter))]

Reply:
probably you meant to add this to the reservationStatus file?

Reply:
yep, my bad

Reply:
added